### DISCUSSION TITLE
will windows 11 23H2 support mirrored network

### DISCUSSION THREAD

**Discussion by fengxx** [microsoft/WSL#10599] [General]
https://devblogs.microsoft.com/commandline/windows-subsystem-for-linux-september-2023-update/
It seems mirrored network works under VPN, the blog mentioned it is available since 22H2 22621.2359 build, will the feature be rolled out to windows 11 23H2 stable version?
I tried on 
WSL version: 2.0.4.0
Windows version: 10.0.22631.2359

got error Wsl/Service/CreateInstance/CreateVm/ConfigureNetworking/0x800704db, and event logger shows
```
{ "hresult":-2147023653, "fileName":"onecore\\vm\\dv\\net\\hns\\service\\core\\networkentitymanager.cpp", "lineNumber":1771, "module":"hostnetsvc.dll", "failureType":0, "message":"vfpext driver is either not installed or is not enabled.", "threadId":18856, "callContext":"\\HnsRpcV2_Enumerate", "originatingContextId":3, "originatingContextName":"HnsRpcV2_Enumerate", "originatingContextMessage":"", "currentContextId":3, "currentContextName":"HnsRpcV2_Enumerate", "currentContextMessage":"", "failureId":16, "failureCount":9, "function":"" }
```

**Comment by nlvw** [comment]
I'm on Win 11 insider release preview (23H2 22631.2361) and the mirror feature works there for me.  I hope it hits stable, non-insider, channel soon  as well.

The mirroring has worked great for me as it not only mirrors your wifi/lan IP but also any hyper-v switches/networks you have setup.  I did test with Cisco Anyconnect at work and while it didn't work seamlessly I just had to do a `wsl --shutdown` and then relaunch my wsl distro and all the routing worked correctly.

**Comment by fengxx** [comment]
found out `vfpext driver` is provided by VirtualMachinePlatform feature
`dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart`
